src/main/scala/com/codacy/client/bitbucket/PullRequest.scala

```diff
@@ -57,7 +57,7 @@ object PullRequest {
       (__ \ "source" \ "commit" \ "hash").read[String] and
       (__ \ "destination" \ "repository" \ "full_name").read[String] and
       (__ \ "destination" \ "branch" \ "name").read[String] and
-      (__ \ "destination" \ "commit" \ "hash").read[String] and
+      (__ \ "destination" \ "commit" \ "hash").readNullable[String] and
```

Comment:
Sometimes play json cannot use nullable and needs `read[Option[String]]`.

The behaviour is different. No idea why.

Have you tested this?